src/Raven.Client/Exceptions/ConcurrencyException.cs

```diff
@@ -56,5 +59,40 @@ public ConcurrencyException(string message, Exception inner) : base(message, inn
         /// Actual Change Vector.
         /// </summary>
         public string ActualChangeVector { get; set; }
+
+        /// <summary>
+        /// The Document Id.
+        /// </summary>
+        public string Id;
+
+        /// <summary>
+        /// Cluster Concurrency violations info.
+        /// </summary>
+        public Conflict[] ClusterConcurrencyViolations;
```

Comment:
This is filled only for cluster wide tx concurrency violations right?
wonder if it would be better to have `ClusterWideConcurrencyException : ConcurrencyException` instead.
@ppekrol please advice :-)

Reply:
and `ClusterConcurrencyViolations` -> `ConcurrencyViolations`

Reply:
done